documentation/MIGRATION_GUIDE.md

```diff
@@ -172,6 +195,11 @@ In latest version (`sdk/resourcemanager/**/arm**`), we use `arm.ClientOptions.Pe
 Similar to the customized policy, there are changes regarding how the custom HTTP client is configured as well. You can now use the `arm.ClientOptions.Transport` option in `github.com/Azure/azure-sdk-for-go/sdk/azcore/arm` package to use your own implementation of HTTP client and plug in what they need into the configuration.  The HTTP client must implement the `policy.Transporter` interface.
 
 **Previous version (`services/**/mgmt/**`)**
+
+```go
+import "github.com/Azure/azure-sdk-for-go/services/resources/mgmt/2020-10-01/resources"
```

Comment:
In previous version, people could use APIs from different API version (even preview+ stable), I don't think this is supported in new version, right?

Reply:
Yes. I've added some notes for this.